e2e-tests/testdata/echo_client/.eslintrc.cjs

```diff
@@ -5,6 +5,12 @@ module.exports = {
     plugins: [
         '@typescript-eslint',
     ],
+    env: {
+        browser: true,
+        es6: true,
+        node: true,
+        commonjs: true
+    },
```

Comment:
What does this do?

Reply:
This tells ESLint to not throw any errors based on which APIs are available in which environment. By telling ESLint this is a library that will work with ES6, in the browser, and in node, then it understands to not throw for things like the `fetch` API being missing in older versions of node.